prepare_stmt.go

```diff
@@ -17,16 +20,52 @@ type Stmt struct {
 }
 
 type PreparedStmtDB struct {
-	Stmts map[string]*Stmt
+	Stmts StmtStore
 	Mux   *sync.RWMutex
 	ConnPool
 }
 
-func NewPreparedStmtDB(connPool ConnPool) *PreparedStmtDB {
+const DEFAULT_MAX_SIZE = (1 << 63) - 1
```

Comment:
Could we relocate the majority of the following code to an internal package, such as internal/stmt_store, to prevent the exposure of additional structs or methods?

Reply:
cause circular dependency